src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  border-radius: 0.5px;
```

Comment:
The border-radius value of `0.5px` is quite small and might not be noticeable. Consider using a larger value if you want the rounded corners to be more visible.